scripts/internal/windows_build_common.py

```diff
@@ -4,7 +4,7 @@
 import os
 import shutil
 
-DEFAULT_PY_ENVS = ["37-x64", "38-x64", "39-x64", "310-x64"]
```

Comment:
Will require update to add 3.11 similar to https://github.com/scikit-build/scikit-ci-addons/pull/95/files

Reply:
Added https://github.com/scikit-build/scikit-ci-addons/pull/101 to support.

Reply:
https://github.com/scikit-build/scikit-ci-addons/pull/101 is merged, thank you @jcfr for your help 👍